cms/__init__.py

```diff
@@ -137,6 +139,20 @@
     LANG_PASCAL: ".o",
 }
 
+COMPILED = 1
+HALF_COMPILED = 3
+INTERPRETED = 5
```

Comment:
A lot of a yolo actually, if people feel it's important I can run proper timing tests and attempt to compare them all properly (though what that means is unclear, are we catering to a beginner interpreted language programmer who has no clue of how much overhead things have or someone who nows what they're doing?), but I expect results to be "close enough" based on previous experience